docs/docs/genai/eval-monitor/scorers/third-party.mdx

```diff
@@ -0,0 +1,207 @@
+import { APILink } from "@site/src/components/APILink";
+import TileCard from '@site/src/components/TileCard';
+import TilesGrid from '@site/src/components/TilesGrid';
+import { Bot, GitBranch, Shield } from "lucide-react";
+
+# Third-party Scorers
+
+MLflow integrates with popular third-party evaluation frameworks, allowing you to leverage their specialized metrics within MLflow's evaluation workflow. This provides access to battle-tested evaluation metrics while maintaining a consistent MLflow interface.
```

Comment:
We don't know how battle-tested 3p framework judges are. Could we just say "access to third party library's evaluation metrics while ..."